### PR TITLE
completes task 9

### PR DESCRIPTION
**Pull request by Thomas-Smith0176**
completes DELETE /api/comments/comment_id endpoint

### CODE REVIEW

File: controllers/comments-controllers.js [hunk 1]
```diff
@@ -17,6 +17,14 @@ exports.getComments = (req, res, next) => {
     .catch(next);
 };
 
+exports.removeComment = (req, res, next) => {
+    const { comment_id } = req.params
+    return deleteComment(comment_id)
+    .then(()=> {
+        res.status(204).send({})
```
Comment (mezzDavies) [inline controllers/comments-controllers.js:24]: No need to send anything for this endpoint .send invoked with nothing is fine.

**Review by mezzDavies — COMMENTED** [review]
Looking good but you need an additional test - what if I try to delete a comment that doesn't exist eg comment with an id of 99999. Please add this test and functionality. 😄

File: controllers/comments-controllers.js [hunk 2]
```diff
@@ -17,6 +17,17 @@ exports.getComments = (req, res, next) => {
     .catch(next);
 };
 
+exports.removeComment = (req, res, next) => {
+    const { comment_id } = req.params
+    return deleteComment(comment_id)
+    .then((comment)=> {
+        if (comment.length === 0) {
```
Comment (mezzDavies) [inline controllers/comments-controllers.js:24]: A comment is an object so doesn't have a length property 🤔

File: models/comments-models.js [hunk 3]
```diff
@@ -12,6 +12,16 @@ exports.selectCommentsByArticleId = (articleId) => {
     });
 };
 
+exports.deleteComment = (comment_id) => {
```
Comment (mezzDavies) [inline models/comments-models.js:15]: Your model and controller are named the wrong way round. The convention is the controller should include the method (delete in this case) and the model should use a verb that means the same thing eg "remove".

**Review by mezzDavies — APPROVED** [review]
Left a couple of bits for you to thik about but this approved for merge 👍

File: controllers/comments-controllers.js (hunk 2 above)
Comment (Thomas-Smith0176) [reply controllers/comments-controllers.js:24]: sorry this is my code just being unnecessarily confusing haha,  comment is response.rows here so i'm just checking if its an empty array or not

**Pull request merged**